src/adapters/slack/utils.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+class SlackUtils {
```

Comment:
Please explain. I'm not sure why we should avoid using Classes. This seems to be a Javascript sort of design consideration. In most languages I place functionality under the heading of a Class in order to group related behaviors, whether static or not. What would it look like for SlackUtils to not be a Class but still be usable throughout the app?

Reply:
Classes maintain state. If you do not have a state, it's generally better to have stateless functions, because javascript is an event driven language and it's kinda hard to pass state around.